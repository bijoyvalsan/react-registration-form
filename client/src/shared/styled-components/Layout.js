import styled from 'styled-components';

export const PageLayout = styled.div`
    background-color: #f0f0f0;
    height: 100%;
`;