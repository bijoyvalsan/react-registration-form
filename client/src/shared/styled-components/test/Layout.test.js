import React from 'react';
import 'jest-styled-components';
import renderer from 'react-test-renderer';

import { PageLayout } from '../Layout';

describe('Styled Components: PageLayout', () => {
  it('renders correctly - PageLayout Style Component', () => {
    const tree = renderer.create(<PageLayout>Testing</PageLayout>).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
