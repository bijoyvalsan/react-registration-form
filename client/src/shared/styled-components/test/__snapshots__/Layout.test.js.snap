// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Styled Components: PageLayout renders correctly - PageLayout Style Component 1`] = `
.c0 {
  background-color: #f0f0f0;
  height: 100%;
}

<div
  className="c0"
>
  Testing
</div>
`;
